spanner/kokoro/integration.sh

```diff
@@ -142,11 +143,11 @@ elif [[ $KOKORO_JOB_NAME == *"nightly"* ]]; then
   fi
 fi
 
-#if [[ $KOKORO_BUILD_ARTIFACTS_SUBDIR = *"continuous"* ]] || [[ $KOKORO_BUILD_ARTIFACTS_SUBDIR = *"nightly"* ]]; then
-#  chmod +x $KOKORO_GFILE_DIR/linux_amd64/flakybot
-#  $KOKORO_GFILE_DIR/linux_amd64/flakybot -logs_dir=$GOCLOUD_HOME \
-#    -repo=googleapis/google-cloud-go \
-#    -commit_hash=$KOKORO_GITHUB_COMMIT_URL_google_cloud_go
-#fi
+if [[ $KOKORO_BUILD_ARTIFACTS_SUBDIR = *"continuous"* ]] || [[ $KOKORO_BUILD_ARTIFACTS_SUBDIR = *"nightly"* ]]; then
+  chmod +x $KOKORO_GFILE_DIR/linux_amd64/flakybot
```

Comment:
@rahul2393 
Do we need to enable flaky bot for other environments?

Reply:
currently Java has not yet enabled Flaky bot for error reporting. So following the same and temporarily commenting the code.